### PR TITLE
Update isolated-vm bundler

### PR DESCRIPTION
**Pull request by adrinr**
## Description
Update isolated-vm bundle after the string-templates migration from cjs to esm.

## Feature branch env
[Feature Branch Link](http://fb-budi-8113-fix-isolated-vm-bundler.fb.qa.budibase.net)

### CODE REVIEW

File: packages/server/src/jsRunner/bundles/snippets.ivm.bundle.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
-"use strict";var snippets=(()=>{var u=Object.create;var n=Object.defineProperty;var a=Object.getOwnPropertyDescriptor;var h=Object.getOwnPropertyNames;var x=Object.getPrototypeOf,C=Object.prototype.hasOwnProperty;var l=(i,e)=>()=>(e||i((e={exports:{}}).exports,e),e.exports),W=(i,e)=>{for(var p in e)n(i,p,{get:e[p],enumerable:!0})},f=(i,e,p,r)=>{if(e&&typeof e=="object"||typeof e=="function")for(let t of h(e))!C.call(i,t)&&t!==p&&n(i,t,{get:()=>e[t],enumerable:!(r=a(e,t))||r.enumerable});return i};var d=(i,e,p)=>(p=i!=null?u(x(i)):{},f(e||!i||!i.__esModule?n(p,"default",{value:i,enumerable:!0}):p,i)),g=i=>f(n({},"__esModule",{value:!0}),i);var s=l((D,o)=>{o.exports.iifeWrapper=i=>`(function(){
```
Comment (adrinr) [inline packages/server/src/jsRunner/bundles/snippets.ivm.bundle.js:1]: I see quite a lot of changes in here. I tested it and it works fine for me, could you double-check it @aptkingston ?

Comment (aptkingston) [reply packages/server/src/jsRunner/bundles/snippets.ivm.bundle.js:1]: Will do!

Comment (aptkingston) [reply packages/server/src/jsRunner/bundles/snippets.ivm.bundle.js:1]: Yep seems totally fine to me - snippets still work in isolated-vm as expected.

**Review by aptkingston — APPROVED** [review]
LGTM!

**Comment by aptkingston** [comment]
Actually - its not part of this PR but there's an error on line 60 of the helpers `list.ts` - it should be
```js
  helpers = {
    ...helpers,
    ...addedHelpers,
  }
```
instead of 
```js
  helpers = {
    ...helpers,
    addedHelpers,
  }
```
Currently the date and duration helpers are broken as mentioned in https://github.com/Budibase/budibase/issues/13333.

Do you want to include that small change as part of this PR?

File: packages/string-templates/src/helpers/list.ts [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
-  string,
-  comparison,
-  object,
-  regex,
-  uuid,
-  // @ts-expect-error
-} from "@budibase/handlebars-helpers"
+/* eslint-disable local-rules/no-budibase-imports */
+// @ts-expect-error
+import math from "@budibase/handlebars-helpers/lib/math"
```
Comment (mike12345567) [inline packages/string-templates/src/helpers/list.ts:5]: should we comment this to explain what is going on here?

**Review by mike12345567 — APPROVED** [review]
LGTM - might just be worth commenting the list of imports, as its quite confusing why the change from importing all of handlebars helpers is needed.

**Pull request merged**